src/api/dirmonitor/fsevents.c

```diff
@@ -57,57 +55,42 @@ static void stop_monitor_stream(struct dirmonitor_internal* monitor) {
 }
 
 
-void deinit_dirmonitor(struct dirmonitor_internal* monitor) {
-  stop_monitor_stream(monitor);
-}
+void deinit_dirmonitor(struct dirmonitor_internal *monitor) { stop_monitor_stream(monitor); }
 
 
-static void stream_callback(
-  ConstFSEventStreamRef streamRef,
-  void* monitor_ptr,
-  size_t numEvents,
-  void* eventPaths,
-  const FSEventStreamEventFlags eventFlags[],
-  const FSEventStreamEventId eventIds[]
-)
-{
-  if (numEvents <= 0) {
-    return;
-  }
+static void stream_callback(ConstFSEventStreamRef         streamRef,
+                            void                         *monitor_ptr,
+                            size_t                        numEvents,
+                            void                         *eventPaths,
+                            const FSEventStreamEventFlags eventFlags[],
+                            const FSEventStreamEventId    eventIds[]) {
```

Comment:
I'm not sure I like this. What other formatting alternatives are there?

Reply:
It's pretty this or align the parameters on a newline.